force-app/main/default/lwc/serviceScheduleCreatorWrapper/__tests__/serviceScheduleCreatorWrapper.test.js

```diff
@@ -21,22 +21,6 @@ describe("c-service-schedule-creator", () => {
         });
     });
 
-    it("modal appears in non-experience-cloud context and element is accessible", () => {
-        element.isCommunity = false;
-        document.body.appendChild(element);
-
-        return global.flushPromises().then(async () => {
-            const modal = element.shadowRoot.querySelector("c-modal");
-
-            // Modal will only display with a spinner loaded
-            expect(modal).not.toBeNull();
-            modal.dispatchEvent(new CustomEvent("dialogclose"));
-
-            // TODO: Validate accessibility when each step is loads.
-            global.isAccessible(element);
-        });
-    });
-
```

Comment:
We'll have to replace this test using the new modal functionality.  It's unclear in the short term how this is done - more research required.